### ISSUE TITLE
Keep the sequence after mathematical calculations

### ISSUE THREAD

**Issue by uvchik** [oemof/oemof#476]
I have two sequence attributes (`a` and `b`) and want to multiply them to get `c`. Is there a way that `c` is a sequences as well?

**Comment by henhuy** [comment]
Yes - that should be the normal case.
I will add this functionalit

**Comment by henhuy** [comment]
Sorry, I pressed the wrong button...

**Comment by uvchik** [comment]
An example that shows the problem can be found in the `ExtractionTurbineCHP` class.
A loop over all TIMESTEPS is used to calculate the `main_flow_loss_index`, just in case the sequence is a real time series. This produces a real time series even if all attributes are scalars (pseudo lists).

```python
n.main_flow_loss_index = [
    (n.conversion_factor_full_condensation_sq[t] -
     n.conversion_factors[m.es.groups[n.main_output.label]][t]) /
     n.conversion_factors[m.es.groups[n.tapped_output.label]][t]
    for t in m.TIMESTEPS]
```
 [see full example](https://github.com/oemof/oemof/blob/dev/oemof/solph/components.py#L935)

It would be nice if it would work like this:

```python
n.main_flow_loss_index = (
    (n.conversion_factor_full_condensation_sq -
     n.conversion_factors[m.es.groups[n.main_output.label]]) /
     n.conversion_factors[m.es.groups[n.tapped_output.label]]
    )
```

If it is not possible like this we could test whether one of the attributes is a list and loop only if a real list is present.

**Comment by uvchik** [comment]
Closed in favour of #477.
